tests/unit/test_tag_facebook_pixel.py

```diff
@@ -41,46 +42,45 @@ class FacebookPixelTagTestCase(TagTestCase):
 
     def test_head_tag(self):
         html = self.render_tag('facebook_pixel', 'facebook_pixel_head')
-        self.assertEqual(expected_head_html, html)
+        assert expected_head_html == html
 
     def test_head_node(self):
         html = FacebookPixelHeadNode().render(Context({}))
-        self.assertEqual(expected_head_html, html)
+        assert expected_head_html == html
 
     def test_body_tag(self):
         html = self.render_tag('facebook_pixel', 'facebook_pixel_body')
-        self.assertEqual(expected_body_html, html)
+        assert expected_body_html == html
 
     def test_body_node(self):
         html = FacebookPixelBodyNode().render(Context({}))
-        self.assertEqual(expected_body_html, html)
+        assert expected_body_html == html
 
     def test_tags_take_no_args(self):
-        self.assertRaisesRegex(
-            TemplateSyntaxError,
-            r"^'facebook_pixel_head' takes no arguments$",
-            lambda: (Template('{% load facebook_pixel %}{% facebook_pixel_head "arg" %}')
-                     .render(Context({}))),
-        )
-        self.assertRaisesRegex(
-            TemplateSyntaxError,
-            r"^'facebook_pixel_body' takes no arguments$",
-            lambda: (Template('{% load facebook_pixel %}{% facebook_pixel_body "arg" %}')
-                     .render(Context({}))),
-        )
+        template = '{%% load facebook_pixel %%}{%% facebook_pixel_%s "arg" %%}'
+        with pytest.raises(TemplateSyntaxError, match="'facebook_pixel_head' takes no arguments"):
+            Template(template % "head").render(Context({}))
```

Comment:
Oh my. Isn't this much better than a lambda :heart:

Reply:
Turn the `"'facebook_pixel_head' takes no arguments"` into a variable (e.g. `error_message`) then it will read, "match equals error message", which will be even nicer! :smiley: :heart_eyes: